install/install.sh

```diff
@@ -41,13 +41,15 @@ usage() {
     * Specify a database management system. Valid options include: postgres, sqlite (if not specified, the default is to use sqlite).
       If the database management system is specified to use postgres, the following extra variables must also be set: QPC_DBMS_HOST, QPC_DBMS_PASSWORD
          -e QPC_DBMS=sqlite
+    * Specify if you would like to use external preconfigured PostgresSQL.
+         -e EXTERNAL_POSTGRES=true
     * If using postgres, specify the db name (if not specified the default value is postgres):
          -e QPC_DBMS_DATABASE=postgres
     * If using postgres, specify the db user (if not specified the default value is postgres):
          -e QPC_DBMS_USER=postgres
-    * If using postgres, you must specify the db password:
+    * If using postgres, you must specify the db password if EXTERNAL_POSTGRES = true (otherwise default value is password):
          -e QPC_DBMS_PASSWORD=password
-    * If using postgres, you must specify the db host:
+    * If using postgres, you must specify the db host if EXTERNAL_POSTGRES = true (otherwise the default is set to the postgres container):
```

Comment:
Along the same lines as above, what happens if they are using the internal postgres but they set a QPC_DBMS_HOST?  If this will break our flow, maybe it should be clear that they can't set this value if not using an external DB. Conversely, if they are using an external DB and fail to provide the QPC_DBMS_HOST 
but it does not match the default, will it bomb out? Or will it silently not work? What do we want to happen? 

 (This question is also for Kevan not just you. )

Reply:
Perhaps these should be internal passwords/host that don't match the same naming convention as the external variables, so that we don't have to worry about users leaving the defaults for the internal postgres?

ie. When I allowed users to pass in external env variables, if they failed to specify a host or password, it automatically used sqlite to avoid a silent inability to reach the database. << not saying this was a good behavior, but it was in the documentation. We should decide how we want to handle this